### ISSUE TITLE
search results - images with no alt value

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#114]
When using the search feature, WAVE alerts to images not having an alt value/text. 

https://rrchnm.org/?s=news
https://rrchnm.org/?s=annual+report

**Comment by kimisgold** [comment]
With https://github.com/chnm/rrchnm-wordpress/commit/82a4a4df10ee8eda1eafe1ade99201bc19416645, this should now be fixed.

**Comment by IdaAkiwumi** [comment]
WAVE is alerting to these 4 images (on searching for news) having nearby images with same alt text. Something similar is happening on the other search (annual report). I wonder if it's the repeated alt text on the text link?
![Screen Shot 2020-10-19 at 10 59 59 AM](https://user-images.githubusercontent.com/618307/96468935-a739a080-11fa-11eb-9b4a-dd85cb85b762.png)
![Screen Shot 2020-10-19 at 11 00 14 AM](https://user-images.githubusercontent.com/618307/96468936-a739a080-11fa-11eb-944a-28f5c6739eea.png)
![Screen Shot 2020-10-19 at 11 00 22 AM](https://user-images.githubusercontent.com/618307/96468939-a86acd80-11fa-11eb-9381-ca7b56d37135.png)
![Screen Shot 2020-10-19 at 11 00 31 AM](https://user-images.githubusercontent.com/618307/96468941-a9036400-11fa-11eb-9b1e-e8ad6dc1886a.png)

**Comment by kimisgold** [comment]
The fix in https://github.com/chnm/rrchnm-wordpress/commit/23bd3670bd681bfd836637aa3999c434e33ea92e should also apply here.
